### PR TITLE
Feature-Back to Top

### PR DESCRIPTION
**Pull request by RachelAnnB**
-add content, styles, and js for the back to top button
-add html and js for smooth scrolling
-add html just for the owl fun fact
-made small tweaks overall

### CODE REVIEW

File: src/scss/_back-to-top.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#return-to-top {
+  position: fixed;
+  bottom: 1.25em; //20px
+  right: 1.25em; //20px
+  background: rgb(0, 0, 0);
```
Comment (aspencer116) [inline src/scss/_back-to-top.scss]: Just curious, is there a reason you are using rgb colors instead of typical hex?

Comment (RachelAnnB) [inline src/scss/_back-to-top.scss]: Nope, just missed that color after grabbing this from my codepen. Good catch!

Comment (aspencer116) [inline src/scss/_back-to-top.scss]: Ok, I was able to pull down your project and view it in my browser (thanks 100% to Zastrow). The code looks good as far as I can. Love the smooth scrolling you have, I might use some of that if you don't mind. The "back to top" button might need to have it's z-index adjusted, it seems to be going behind a few items on your page.

Comment (RachelAnnB) [inline src/scss/_back-to-top.scss]: Sure, go for it! I forked a pen on codepen, so here's how I originally did the smooth scrolling: http://codepen.io/rachelbroering/pen/XKxYAA. I'll also make sure to check out that button. Would you recommend any other changes?

Comment (aspencer116) [inline src/scss/_back-to-top.scss]: That's all I can see for now!

Comment (RachelAnnB) [inline src/scss/_back-to-top.scss]: Awesome, thanks!

**Pull request merged**